web/docs/_TypescriptServerNote.md

```diff
@@ -0,0 +1,7 @@
+:::caution LSP Problems
+
+If you are using TypeScript, your editor may sometimes report type and import errors even while `wasp start` is running.
+
+This happens when the TypeScript Language Server gets out of sync with the current code. If you're using VS Code, you can manually restart the language server by opening the command palette and selecting _"TypeScript: Restart TS Server."_
```

Comment:
I'd maybe mention that the "command palette" is opened by CTRL + SHIFT + P or CMD + SHIFT + P on Mac.